invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/fields/CreatibutorsField/CreatibutorsModal.js

```diff
@@ -384,9 +384,9 @@ export class CreatibutorsModal extends Component {
     const selectedSuggestion = selectedSuggestions[0].extra;
     this.setState(
       {
-        organizationIdentifiers: selectedSuggestion.identifiers.map(
-          (identifier) => identifier.identifier
-        ),
+        organizationIdentifiers: selectedSuggestion.identifiers
+          .filter((identifier) => identifier.scheme !== "grid") // Filtering out unsupported org scheme i.e. "grid"
```

Comment:
I'm still thinking it would be good to reference `RDM_RECORDS_PERSONORG_SCHEMES` here for better discoverability.